core/src/main/scala/org/apache/spark/sql/SnappyBaseParser.scala

```diff
@@ -397,6 +397,8 @@ object SnappyParserConsts {
   final val WHERE: Keyword = reservedKeyword("where")
   final val WITH: Keyword = reservedKeyword("with")
   final val FUNCTION: Keyword = reservedKeyword("function")
+  final val POLICY: Keyword = reservedKeyword("policy")
```

Comment:
This is not reserved in any of the dbs including Postgres (e.g. see https://www.petefreitag.com/tools/sql_reserved_words_checker/?word=POLICY). Can this not be made nonReservedKeyword?

Reply:
ok. will do it